pkg/operator/interfaces.go

```diff
@@ -1,4 +1,4 @@
-package operatorclient
```

Comment:
note that this was just to prevent package import clashes with the generated client. i can move it back if this is more idiomatic.

Reply:
I would move it back

Reply:
ack, ok